emc2/plotting/subcol_display.py

```diff
@@ -433,26 +458,200 @@ def plot_subcolumn_mean_profile(self, variable, time, pressure_coords=True, titl
         """
 
         ds_name = [x for x in self._arm.keys()][0]
-        my_ds = self._arm[ds_name].sel(time=time, method='nearest')
+        x_variable = self.model.time_dim
+        if type(time) is tuple:
+            time = np.array(time)
+        if time.size == 1:
```

Comment:
The test is failing here at time.size == 1. It says that .size is not a property of string. You'll want to insert a clause here to also handle the case where time is a string.

Reply:
Resolved now, thanks!